### ISSUE TITLE
DefaultSHKConfigurator.h file not found error in ShareKit Demo App

### ISSUE THREAD

**Issue by saad352** [ShareKit/ShareKit#1056]
I am using Cocoapods for the first time.I downloaded the ShareKitDemoApp and intalled the pods and it was successful but when i built the project it gives me an error in ShareKitDemoConfigurator.h that 'DefaultSHKConfigurator.h' is missing.

Am i missing something? or this is not the way of doing it.
![screen shot 2016-01-31 at 8 01 42 am](https://cloud.githubusercontent.com/assets/11598940/12703158/f7f99100-c7f0-11e5-8b7a-1baee48576bc.png)

**Comment by VilemKurz** [comment]
Have you opened the demo *workspace* file?

**Comment by saad352** [comment]
Yes i am using demo xcworkspace. But i can not understand what you mean by running ShareKit's Cocoapods Demo. Yes there are two targets as i see
![screen shot 2016-01-31 at 10 12 02 am](https://cloud.githubusercontent.com/assets/11598940/12703893/2f87f622-c803-11e5-8605-341f57441d6b.png)
Are you talking about these targets? if yes then how to run the Cocoapods demo?
if not what are the targets you are talking about?

**Comment by saad352** [comment]
UPDATE: I managed to run the ShareKit Demo App (CocoaPods). But now i am getting more error which are mostly parse error. Here is the screen shot
![screen shot 2016-02-01 at 8 29 19 am](https://cloud.githubusercontent.com/assets/11598940/12723455/68131df8-c8be-11e5-958d-9433a3ca5e65.png)
I tried to get rid of them but that results in more error

**Comment by VilemKurz** [comment]
tldr old Xcode version
The detailed answer is here http://stackoverflow.com/questions/35137891/error-in-bftaskcompletionsource-h-in-sharekit-ios.

**Comment by saad352** [comment]
Yeah the old version is the problem..Thanks
